gremlin-groovy-test/src/main/java/org/apache/tinkerpop/gremlin/groovy/jsr223/GroovyTranslatorTest.java

```diff
@@ -141,4 +144,52 @@ public void shouldHandleEmptyMaps() {
     public void shouldHaveValidToString() {
         assertEquals("translator[h:gremlin-groovy]", GroovyTranslator.of("h").toString());
     }
+
+    @Test
+    @LoadGraphWith(LoadGraphWith.GraphData.MODERN)
```

Comment:
It doesn't look like you need the "modern" graph data here - you can remove this annotation.

Reply:
Thanks for pointing that out. I have combined the tests.